shader/coarse.wgsl

```diff
@@ -377,6 +384,13 @@ fn main(
                             write_grad(CMD_RAD_GRAD, index, info_offset);
                         }
                     }
+                    // DRAWTAG_FILL_IMAGE
+                    case 0x248u: {
```

Comment:
I had to change those values several times while playing with the encoding and missing one of these actually led to a TDR. So yeah, this is supremely annoying.

Reply:
On the naga side this seems to tracked by https://github.com/gfx-rs/naga/issues/2106. I'm putting a mention here and I'll ping the naga issue as well as this would be a huge quality of life improvement (I've wanted this in other projects as well).

Reply:
Thanks for the link. This will hopefully make the cut for the next release.